grafonnet/text.libsonnet

```
@@ -20,5 +36,8 @@
       content: content,
       [if description != null then 'description']: description,
       datasource: datasource,
+      [if repeat != null then 'repeat']: repeat,
+      [if repeat != null then 'repeatDirection']: repeatDirection,
+      [if repeat != null then 'repeatMaxPerRow']: repeatMaxPerRow,
```

Looks like this property is actually "maxPerRow". Just checked on 7.0.0 and 7.1.4.

```suggestion
      [if repeat != null then 'maxPerRow']: repeatMaxPerRow,
```

"repeatMaxPerRow" is a bit more human-friendly though so I think leaving the argument name alone is fine.